### PR TITLE
AFNI update

### LINKED ISSUE #651: 3dSkullStrip errors out inside screen
Running the latest mriqc (0.9.9) within Docker, I get the following error if it is run inside of `screen`, which doesn't forward X11. Running it outside of `screen` does not produce the error, but also doesn't seem to need X11 forwarding. The crash file is:

````
Node: workflow_enumerator.anatMRIQCT1w.AFNISkullStripWorkflow.skullstrip
Working directory: /usr/local/src/mriqc/work/workflow_enumerator/anatMRIQCT1w/AFNISkullStripWorkflow/_in_file_..data..sub-11..ses-day2..anat..sub-11_ses-$

Node inputs:

args = <undefined>
environ = {}
ignore_exception = False
in_file = /usr/local/src/mriqc/work/workflow_enumerator/anatMRIQCT1w/AFNISkullStripWorkflow/_in_file_..data..sub-11..ses-day2..anat..sub-11_ses-day1_T1w.$
num_threads = 1
out_file = <undefined>
outputtype = NIFTI_GZ
terminal_output = stream

Traceback (most recent call last):
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/plugins/multiproc.py", line 51, in run_node
    result['result'] = node.run(updatehash=updatehash)
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/engine/nodes.py", line 407, in run
    self._run_interface()
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/engine/nodes.py", line 517, in _run_interface
    self._result = self._run_command(execute)
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/pipeline/engine/nodes.py", line 650, in _run_command
    result = self._interface.run()
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/interfaces/base.py", line 1086, in run
    runtime = self._run_interface(runtime)
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/interfaces/afni/base.py", line 129, in _run_interface
    return super(AFNICommandBase, self)._run_interface(runtime)
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/interfaces/base.py", line 1587, in _run_interface
    self.raise_exception(runtime)
  File "/usr/local/miniconda/lib/python3.6/site-packages/niworkflows/nipype/interfaces/base.py", line 1526, in raise_exception
    **runtime.dictcopy()))
RuntimeError: Command:
3dSkullStrip -input /usr/local/src/mriqc/work/workflow_enumerator/anatMRIQCT1w/AFNISkullStripWorkflow/_in_file_..data..sub-11..ses-day2..anat..sub-11_ses$
Standard output:

Standard error:
freeglut (3dSkullStrip): failed to open display ':1286930384'

Return code: 1
````

### PR DESCRIPTION
**Pull request by chrisfilo**
Closes #651 and #615.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -34,12 +32,33 @@ ENV FSLDIR=/usr/share/fsl/5.0 \
     LD_LIBRARY_PATH=/usr/lib/fsl/5.0:$LD_LIBRARY_PATH \
     FSLTCLSH=/usr/bin/tclsh \
     FSLWISH=/usr/bin/wish \
-    AFNI_MODELPATH=/usr/lib/afni/models \
-    AFNI_IMSAVE_WARNINGS=NO \
-    AFNI_TTATLAS_DATASET=/usr/share/afni/atlases \
-    AFNI_PLUGINPATH=/usr/lib/afni/plugins \
```
Comment (oesteban) [inline Dockerfile]: Aren't these variables needed anymore (after the installation of AFNI)

File: mriqc/__about__.py [hunk 2]
```diff
@@ -71,7 +71,7 @@
     'nipy',
     'statsmodels',
     'versioneer',
-    'xvfbwrapper',
+    'xvfbwrapper'
```
Comment (oesteban) [inline mriqc/__about__.py]: Please add the trailing comma so this file does not show differences in two lines when adding dependencies.

File: mriqc/workflows/utils.py [hunk 3]
```diff
@@ -160,3 +162,16 @@ def slice_wise_fft(in_file, ftmask=None, spike_thres=3., out_prefix=None):
     np.savetxt(out_spikes, spikes_list, fmt=b'%d', delimiter=b'\t', header='TR\tZ')
 
     return len(spikes_list), out_spikes, out_fft
+
+def get_fwhmx():
+    fwhm_args = {"combine": True,
+                 "detrend": True}
+    afni_version_tpl = afni.Info.version()
+    afni_version = StrictVersion(".".join([str(afni_version_tpl[0]),
```
Comment (oesteban) [inline mriqc/workflows/utils.py]: What about
```Python
afni_version = StrictVersion('%s.%s.%s' % afni.Info.version())
```

File: Dockerfile (hunk 1 above)
Comment (chrisfilo) [reply Dockerfile]: It does not seem they were ever needed.

File: Dockerfile [hunk 4]
```diff
@@ -10,7 +10,6 @@ RUN apt-get update && \
                     curl \
                     bzip2 \
                     ca-certificates \
-                    xvfb  \
```
Comment (oesteban) [inline Dockerfile]: I just spotted this, sorry for being late. Why do we keep the python `xvfbwrapper` but stop installing xvfb?

Comment (chrisfilo) [reply Dockerfile]: We do install it. I moved it to the afni block below.

**Comment by chrisfilo** [comment]
When this is merged I recommend making a new release.

**Pull request merged**